projects/com.oracle.truffle.llvm.parser.base/src/com/oracle/truffle/llvm/parser/base/model/enums/Linkage.java

```diff
@@ -46,7 +46,8 @@
     AVAILABLE_EXTERNALLY,
     LINKERPRIVATE,
```

Comment:
I know that the enum names are existing code, but could you change the enums to `LINK_ONCE_ODR`, `LINKER_PRIVATE`, `LINKER_PRIVATE_WEAK`, `LINK_ONCE_ODR_AUTO_HIDE`, `DLL_IMPORT`, and `DLL_EXPORT`? It would make them easier to read.